### DISCUSSION TITLE
如何配置规则以解决 Edge 浏览器 Reconnecting Workspaces 的问题？

### DISCUSSION THREAD

**Discussion by xuchkang171** [clash-verge-rev/clash-verge-rev#1869] [General]
![image](https://github.com/user-attachments/assets/77d4a312-3997-482c-a296-8b3a214c4da1)

Clash 连接里筛选出 msedge.exe 的连接，没有发现疑似与 Reconnecting Workspaces 问题相关的域名。

**Comment by wonfen** [comment]
自己研究下规则。

**Reply by xuchkang171** [reply]
> 自己研究下规则。

不是规则的问题。后来解决了：把 TUN Mode 关闭，只开 System Proxy 就正常了。

用的是当前的最新版 Version 1.7.7。Clash Core 用的默认的 Mihomo /verge-mihomo 。
